src/map/status.cpp

```diff
@@ -8218,8 +8216,12 @@ static short status_calc_aspd(struct block_list *bl, status_change *sc, bool fix
 		map_session_data* sd = BL_CAST(BL_PC, bl);
 		uint8 skill_lv;
 
-		if (sd && (skill_lv = pc_checkskill(sd, BA_MUSICALLESSON)) > 0)
-			bonus += skill_lv;
+		if (sd) {
```

Comment:
why you don't make something like this:
`if (sd && ((skill_lv = pc_checkskill(sd, BA_MUSICALLESSON)) > 0 || (skill_lv = pc_checkskill(sd, RG_PLAGIARISM)) > 0) bonus += skill_lv` ?
Just curious.